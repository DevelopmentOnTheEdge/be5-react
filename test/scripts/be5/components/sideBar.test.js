import React    from 'react';
import renderer from 'react-test-renderer';
import SideBar     from  '../../../../src/scripts/be5/components/SideBar';
import testUtils      from '../testUtils'

it('snapshot sideBar', () => {
  testUtils.mockBe5();

  const component = renderer.create(
    <SideBar />
  );

  expect(component.toJSON()).toMatchSnapshot();
});