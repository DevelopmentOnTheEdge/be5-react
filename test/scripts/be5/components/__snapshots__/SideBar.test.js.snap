// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot sideBar 1`] = `
<div
  className="side-bar"
>
  <div
    className="menuContainer"
  >
    <input
      className="searchField form-control"
      onChange={[Function]}
      placeholder="Filter..."
      type="text"
      value=""
    />
    <div
      className="menu"
    >
      <div
        className="menuNode"
      >
        <a
          className="rootMenuItem menuItemWithRef"
          href="#!static/welcome.be"
          onClick={[Function]}
          target=""
        >
          Добро пожаловать
        </a>
        <div />
      </div>
      <div
        className="menuNode"
      >
        <span
          className="rootMenuItem menuItemWithoutRef"
        >
          Пользователи
        </span>
        <div />
        <ul>
          <li>
            <div
              className="menuNode"
            >
              <a
                className="menuItem menuItemWithRef"
                href="#!login"
                onClick={[Function]}
                target=""
              >
                Вход
              </a>
              <div />
            </div>
          </li>
        </ul>
      </div>
      <div
        className="menuNode"
      >
        <span
          className="rootMenuItem menuItemWithoutRef"
        >
          Услуги
        </span>
        <div />
        <ul>
          <li>
            <div
              className="menuNode"
            >
              <a
                className="menuItem menuItemWithRef"
                href="#!table/users/TestQuery"
                onClick={[Function]}
                target=""
              >
                Test query
              </a>
              <div />
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="menuFooter"
  >
    <img
      src={Object {}}
    />
  </div>
</div>
`;
